client/src/App.jsx

```diff
@@ -1,50 +1,76 @@
-const React = require('react');
-const ReactDOM = require('react-dom');
-const $ = require('jquery');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import FiveBooks from './components/FiveBooks.jsx';
 
 class App extends React.Component {
   constructor(props) {
-    super(props)
+    super(props);
 
     this.state = {
-      bookId: 0
-    }
+      authorInfo: {},
+    };
 
-    this.handleChange = this.handleChange.bind(this);
-    this.handleSubmit = this.handleSubmit.bind(this);
+    // this.handleChange = this.handleChange.bind(this);
+    // this.handleSubmit = this.handleSubmit.bind(this);
   }
 
-  handleChange (event) {
-    this.setState({bookId: event.target.value}, () => {
-      console.log('this.state.bookId', this.state.bookId);
-    })
-  }
+  // handleChange (event) {
+  //   this.setState({bookId: event.target.value}, () => {
+  //     console.log('this.state.bookId', this.state.bookId);
+  //   })
+  // }
 
-  handleSubmit (event) {
-    $.ajax({
-      url: 'author',
-      method: 'POST',
-      data: {bookId: this.state.bookId },
-      success: () => {
-        console.log('post worked');
-      }
-    })
+  // handleSubmit (event) {
+  //   $.ajax({
+  //     url: 'author',
+  //     method: 'POST',
+  //     data: { bookId: this.state.bookId },
+  //     success: () => {
+  //       console.log('post worked');
+  //     }
+  //   })
+
+  // }
 
+  componentDidMount() {
+    $.ajax({
+      url: '/author',
+      method: 'GET',
+      data: { bookId: 13 },
+      success: (results) => {
+        this.setState(
+          {
+            authorInfo: results,
+          }, () => {
+            console.log('this.state.authorInfo', this.state.authorInfo);
+          });
+      },
+    });
   }
 
   render() {
+    // const listBooks = this.state.authorInfo.titles.map(title => <p>{title}</p>);
+    console.log('this.state.authorInfo.titles', this.state.authorInfo.titles);
     return (
       <div>
-        <form onSubmit={this.handleSubmit}>
-        <input type="text" value={this.state.bookId} onChange={this.handleChange}></input>
-        <button onSubmit={this.handleSubmit}>Get Book</button>
-        </form>
-        <p><a href="/author/">Author</a></p>
-        <p><a href="/five-books-by-author">Five Books By Author</a></p>
-        <p><a href="/book-item-hover-window">Book Item Hover Window</a></p>
+        <h5>ABOUT {this.state.authorInfo.name}</h5>
+
+        <h4>{this.state.authorInfo.name}</h4>
+        <p>{this.state.authorInfo.followers} followers</p>
```

Comment:
This is probably placeholder too, but you'll likely want everything in this component in a `<div>` for styling purposes eventually

Reply:
Yes, placeholder - I'll try to note 'placeholder' instances going forward so it's more clear.